cnf-certification-test/accesscontrol/suite.go

```diff
@@ -444,3 +450,21 @@ func TestOneProcessPerContainer(env *provider.TestEnvironment) {
 		ginkgo.Fail(errMsg)
 	}
 }
+
+func Test1337UIDs(env *provider.TestEnvironment) {
+	const leetNum = 1337
+	var badPods []string
+	for _, put := range env.Pods {
+		ginkgo.By(fmt.Sprintf("checking if pod %s has a securityContext RunAsUser 1337 (ns= %s)", put.Data.Name, put.Data.Namespace))
+		if put.Data.Spec.SecurityContext.RunAsUser != nil && *put.Data.Spec.SecurityContext.RunAsUser == int64(leetNum) {
```

Comment:
I wonder if it wouldn't be better to check the actual UID of the process launched inside the pod as this _PodSecurityContext_ field can be left unspecified. According to the documentation "defaults to user specified in image metadata if unspecified".

Also, there is a similar field inside _container.securityContext_ that takes precedence over the one in _PodSecurityContext_.

Reply:
Cool I'll play around with that.